rest/route/aws.go

```diff
@@ -189,11 +189,18 @@ func (aws *awsSns) handleInstanceInterruptionWarning(ctx context.Context, instan
 			instanceType = stringVal
 		}
 	}
+	existingHostCount, err := host.CountRunningHosts(h.Distro.Id)
+	if err != nil {
+		grip.Error(errors.Wrap(err, "database error counting running hosts by h.Distro.Id"))
```

Comment:
Can you make this a structured error like this?
```go
grip.Error(message.WrapError(err, message.Fields{
    "message": "could not count hosts in distro during instance interruption",
    "host_id": h.Id,
    "distro_id": h.Distro.Id,
    "instance_type": instanceType,
}))
```

It's difficult to trace log messages from plain `errors.Wrap` error logs since they're unstructured log messages. I would avoid grip logging in general without some kind of structured log message fields.